docs/tutorials/HD143006_Part_1.py

```diff
@@ -0,0 +1,211 @@
+# ---
+
+# + nbsphinx="hidden"
+# %matplotlib inline
+
+# + nbsphinx="hidden"
+# %run notebook_setup
+# -
+
+# # HD143006 Tutorial Part 1
+# 
+# Here we are going to do the first step in real world data analysis with MPoL. We will gather the data and then produce a diagnostic image-- the MPoL Dirty Image. This image will have no RML methods applied to it, but it is still a good preliminary checkpoint. We will see the quality of this image and compare it to the DSHARP image created using the CLEAN method ([Andrews et al. 2018](https://ui.adsabs.harvard.edu/abs/2018ApJ...869L..41A/abstract)).
+# 
+# ### Introduction to Regularized Maximum Likelihood (RML) Imaging
+# 
+# Regularized Maximum Likelihood (RML) imaging is a forward modeling methodology. We predict an image then represent it as an array of pixels. This is brought to the visibility domain through a forward Fourier tranform. This is then compared to the measured data to ensure it is a viable predicted image. Due to information loss of the true image when obtaining the measured data, several predicted images- including incorrect ones- will match. To get to our best predicted image, we make new predictions by choosing the most likely (Maximum Likelihood) configuration and favoring specified criterion (Regularized). The likeliness and how well a predicted image meets a certain criterion is mathematically represented in a loss function. We minimize the loss function by changing pixel parameters in the predicted image, giving us the best version of the predicted image.
+# 
+# 
+# 
+# ### Loading Data
+# 
+# Let's load the data as we've done in previous tutorials. We will be examining the Fiducial Images (.fits) and Extracted Visibilities (.npz) of the HD143006 DSHARP survey. The extracted visibilities were calculated from the Final Calibrated Measurement Sets from DSHARP [here](https://mpol-dev.github.io/visread/) using visread.
+# 
+# *You can either download these two files (HD143006_continuum.fits and HD143006_continuum.npz) directly to your working directory, or use astropy to download them during run time.*
+# 
+# 
+
+from astropy.io import fits
+import matplotlib.pyplot as plt
+import numpy as np
+from astropy.utils.data import download_file
+
+# downloading fits file
+fname_F = download_file(
+   'https://almascience.eso.org/almadata/lp/DSHARP/images/HD143006_continuum.fits',
+   cache=True,
+   pkgname='mpol',
+)
+
+# downloading extracted visibilities file
+fname_EV = download_file(
+    'https://zenodo.org/record/4904794/files/HD143006_continuum.npz',
+    cache=True,
+    pkgname='mpol',
+)
+
+# Now that we have the files, let us examine the fits image created by the DSHARP team using the CLEAN algorithm ([Andrews et al. 2018](https://ui.adsabs.harvard.edu/abs/2018ApJ...869L..41A/abstract)). First let's extract our data and then, to properly display the image, we will need to change from units of pixels to arcseconds. To do this, we will calculate the image extent.
```

Comment:
The Andrews et al. citation is for the DSHARP survey in particular, but this reads as though Andrews et al. is the reference for the CLEAN algorithm.

When used to describe the [FITS files](https://en.wikipedia.org/wiki/FITS), the acronym is usually capitalized.